torch/_refs/__init__.py

```diff
@@ -2675,6 +2676,20 @@ def vstack(tensors: TensorSequenceType) -> TensorLikeType:
     return cat(aligned_tensors, 0)
 
 
+# CompositeImplicitAutograd - don't register decomp
+@out_wrapper()
```

Comment:
I don't think that's correct, `unflatten` always returns a view on the input, and `out_wrapper` will always return a copy. cc @mruberry

Reply:
I proposed this as an extension of the operation, but it's clearly wrong. I guess no view function has an `out=` variant for this reason, right? We should just remove `out_wrapper` here. Now, given that the tests are passing, we're missing a test testing for this.

Reply:
there was a test for comparing view relationship on out, probably it's skipped/disabled. Yeah, removing `out` wrapper if original op doesn't have `out` sounds fine to me.